keps/sig-apps/0753-sidecarcontainers.md

```diff
@@ -953,8 +952,9 @@ this. As always, ideas are very welcome :).
 ##### Suggestion
 
 Go for Alternative 1. Seems simple and leaves the cluster in a clean state.
-However, I'd really would like to know what others think.
+However, I really would like to know what others think.
 
+// THIS SEEMS LIKE IT'S IN THE WRONG PLACE
```

Comment:
Oh, you think it is better on the `Alternative 4` part?

Reply:
Well it's talking about having min of 6 seconds to shutdown which seems unrelated to restart policy so I'm not sure the proceeding paragraph is meant to be here

Reply:
Ohhh, I put it on the totally wrong section! Sorry, now I realize :facepalm:

Reply:
Moved it here: https://github.com/kinvolk/kubernetes-enhancements/blob/852b92e4807629a22fc32f46f36c3de56b1b8b51/keps/sig-apps/0753-sidecarcontainers.md#suggestion-1